wiz-admission-controller/templates/_helpers.tpl

```diff
@@ -0,0 +1,79 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "wiz-admission-controller.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "wiz-admission-controller.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "wiz-admission-controller.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "wiz-admission-controller.labels" -}}
+helm.sh/chart: {{ include "wiz-admission-controller.chart" . }}
+{{ include "wiz-admission-controller.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "wiz-admission-controller.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "wiz-admission-controller.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "wiz-admission-controller.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "wiz-admission-controller.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "wiz-admission-controller.certificate" -}}
+{{ .Values.tlsCertificate.tlsCertificate }}
+{{- end }}
+
+{{- define "wiz-admission-controller.secretApiToken" -}}
+{{ include "wiz-admission-controller.fullname" . }}-api-token
+{{- end }}
+
+{{- define "wiz-admission-controller.opaCliParams.policies" -}}
+{{- if .Values.opaWebhook.policies }}
+--policy={{ join " --policy=" .Values.opaWebhook.policies }}
```

Comment:
Isn't it a duplicate of `--policy=` ?

Reply:
I need to format policies to ``--policy=policy1 --policy=policy2` so I need to add the first --policy manually